### DISCUSSION TITLE
Request for Basic Simulation Code Without Experimental Data

### DISCUSSION THREAD

**Discussion by prem-tekkr** [byuflowlab/FLOWUnsteady#193] [General]
Hello ,

I’m using FLOWUnsteady for the first time and facing difficulties writing my own simulation code without having CFD or experimental data. Can anyone please share example code for simulating a wing or propeller that does not rely on CFD or experimental data for validation? I’d appreciate any guidance or resources to help me get started with defining simulation parameters and running the code independently of external data.

**Comment by EdoAlvarezR** [comment]
Hi @prem-tekkr,

The [basic example](https://flow.byu.edu/FLOWUnsteady/examples/propeller-J040/) in the tutorials does not require any input data (CFD nor experiment) for the airfoil lookup tables---it automatically calls XFOIL on the given airfoil sections to generate its own look up table

[https://flow.byu.edu/FLOWUnsteady/examples/propeller-J040/](https://flow.byu.edu/FLOWUnsteady/examples/propeller-J040/)

**Comment by prem-tekkr** [comment]
Hello @EdoAlvarezR ,

Thank you so much for your response.

**1)**
I can able to run a basic wing simulation with my input data. From the .h5 file, I can calculate the lift, but I’m struggling to calculate drag. 

I am trying to calculate lift and drag using circulation via a MATLAB script, but I’m facing issues in obtaining accurate values for Lift (L).

The example [wing code](https://github.com/byuflowlab/FLOWUnsteady/blob/master/examples/wing/wing.jl) giving the outputs CL and CD after the simulation and compares them with experimental results in the terminal. However, I’m encountering errors when I try to remove the experimental data comparison.

Here’s an example of the output I’m getting:
*************************************************************************
END wing2_500/wing2_500_pfield
	2024-09-25T18:26:17.713
*************************************************************************
ELAPSED TIME: 0 hours 24 minutes 13 seconds

Postprocessing...

	 PARAMETER	Experimental	  FLOWUnsteady	  Error
	        CL	     0.2380	                0.31751	         33.41 ﹪
	        CD	     0.0050	                0.02496	         399.18 ﹪

**How can I adjust the code to display only the FLOWUnsteady CL and CD values after the simulation?**

**2)**
I would also like to know **how to add multiple sections to a wing** in FLOWUnsteady. 

The [basic example](https://flow.byu.edu/FLOWUnsteady/examples/wing-4p2aoa/) creates a VLM wing with constant sweep, dihedral, taper ratio, and linear twist between the root and wing tips.

How can I modify the code to create a wing with different sections, such as combining a rectangular wing with a forward sweep section(as shown in the fig
![unnamed](https://github.com/user-attachments/assets/4da9d5e9-3aae-4b7c-9890-cd667a86b710)
)?
